runtime/autoload/provider/clipboard.vim

```diff
@@ -172,6 +172,11 @@ function! s:clipboard.set(lines, regtype, reg) abort
   if jobid > 0
     call jobsend(jobid, a:lines)
     call jobclose(jobid, 'stdin')
+    " xclip does not close stdout,stderr when receiving input via stdin
+    if argv[0] ==# 'xclip'
+      call jobclose(jobid, 'stdout')
+      call jobclose(jobid, 'stderr')
```

Comment:
I don't think stderr should have been closed here.

Reply:
Closing stdout is insufficient.

Reply:
`Test_display_registers` fails again with `target STRING not available` error.

Reply:
https://github.com/neovim/neovim/pull/11661 Take 2.